desktop/src/com/mygdx/crazybomber/desktop/DesktopLauncher1.java

```diff
@@ -0,0 +1,12 @@
+package com.mygdx.crazybomber.desktop;
+
+import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
+import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
+import com.mygdx.crazybomber.CrazyBomber;
+
+public class DesktopLauncher1 {
```

Comment:
what is this class

Reply:
idk ima delete it, thought you left it there for maybe future use
i copy pasted the code in notepad incase we need later or something